spec/core/encoding.md

```diff
@@ -65,6 +65,31 @@ The address is the first 20-bytes of the SHA256 hash of the raw 32-byte public k
 address = SHA256(pubkey)[:20]
 ```
 
+The public key comprised of 32 bytes for one field element (the x-coordinate),
+plus one byte for the parity of the y-coordinate. The first byte depends is a
+0x02 byte if the y-coordinate is the lexicographically largest of the two
+associated with the x-coordinate. Otherwise the first byte is a 0x03. This
```

Comment:
```suggestion
The public key is comprised of 32 bytes for one field element (the x-coordinate),
plus one byte for the parity of the y-coordinate. The first byte is
0x02 if the y-coordinate is the lexicographically largest of the two
associated with the x-coordinate. Otherwise the first byte is 0x03. This
```